### ISSUE TITLE
Unable to train deeplab on tpu

### ISSUE THREAD

**Issue by tantara** [tensorflow/tpu#164]
I have tested tpu in the past month and noticed that it is impossible to use `tf.image.resize_images` in the training phase.

I am trying to train `tpu/models/experimental/deeplab` that use `tf.image.resize_bilinear` at the last `logits` layer, but training looks like stuck in somewhere as follows:
(There is no next result for 4 hours)

```
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:TPU job name worker
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from gs://${BUCKET_NAME}/pretrained/resnet_v1_101/model.ckpt
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Init TPU system
INFO:tensorflow:Start infeed thread controller
INFO:tensorflow:Starting infeed thread controller.
INFO:tensorflow:Start outfeed thread controller
INFO:tensorflow:Starting outfeed thread controller.
INFO:tensorflow:Enqueue next (2000) batch(es) of data to infeed.
INFO:tensorflow:Dequeue next (2000) batch(es) of data from outfeed.
```

I have tested the same code on tpu with both 1.8 and 1.9 version of tensorflow.

If I change it to `tf.layers.conv2d_transpose`, it works well(e.g. loss and other debug messages are shown continuously) although the final result might be wrong
Is there a way to use `tf.image.resize_images` on tpu?
